platform/local_country_file_utils.cpp

```diff
@@ -305,11 +325,19 @@ shared_ptr<LocalCountryFile> PreparePlaceForCountryFiles(CountryFile const & cou
 
 string GetFileDownloadPath(CountryFile const & countryFile, MapOptions file, int64_t version)
 {
+  return GetFileDownloadPath(countryFile, file, version, string());
+}
+
+string GetFileDownloadPath(CountryFile const & countryFile, MapOptions file,
+                           int64_t version, string const & dataDir)
+{
   Platform & platform = GetPlatform();
   string const readyFile = GetFileName(countryFile.GetName(), file, version) + READY_FILE_EXTENSION;
+  string const dir = dataDir.empty() ? platform.WritableDir()
```

Comment:
Could you please create a short inline function, like:
```c++
string GetFullDirPath(string const & dataDir) {
  return dataDir.empty() ? platform.WritableDir() : my::JoinFol...
}
```
And use it without code duplication?